app/src/main/java/com/duckduckgo/app/global/view/ActivityExtension.kt

```diff
@@ -55,7 +57,13 @@ fun Context.fadeTransitionConfig(): Bundle? {
     return config.toBundle()
 }
 
+@Synchronized
```

Comment:
Why the @Syncronized in all these methods?

Reply:
This was more of a concern earlier when the JavaBridge thread would try to change the orientation and the Main thread would enter/exit full screen (locking orientation is not allowed when the page is not full screen). With the updated code, both pathways now end up on Main, so I removed the @synchronized. Do you think we should add something like @MainThread here instead? Basically, I am trying to protect against a scenario where the user gets locked in some orientation even after exiting full screen.